docs/sql-ref-syntax-qry-select-join.md

```diff
@@ -18,5 +18,52 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+A SQL join is used to combine rows from two relations based on join criteria. The following section describes the overall join syntax and the sub-sections cover different types of joins along with examples.
+
```

Comment:
We need a link to this section from the SELECT document, too?
https://github.com/apache/spark/blame/master/docs/sql-ref-syntax-qry-select.md#L78-L85

Reply:
It already has the ```Join relation``` in the ```from_item```. I will add a link to the ```Related Clauses``` section.